src/Symfony/Component/Intl/Tests/DateFormatter/AbstractIntlDateFormatterTest.php

```diff
@@ -398,8 +398,8 @@ public function testFormatWithDateTimeZone()
             $this->markTestSkipped('Only in PHP 5.5+ IntlDateFormatter allows to use DateTimeZone objects.');
         }
 
-        if (defined('HHVM_VERSION_ID')) {
-            $this->markTestSkipped('This test cannot work on HHVM. See https://github.com/facebook/hhvm/issues/5875 for the issue.');
+        if (defined('HHVM_VERSION_ID') || PHP_VERSION_ID <= 50509) {
```

Comment:
ah indeed. For PHP 5.4, we talk about IntlDateFormatter features, not about DateTimeZone limitations. So I agree about keeping like that